### DISCUSSION TITLE
Which roles do I need for the GET_LEDGER_DETAIL_VIEW_DATA report?

### DISCUSSION THREAD

**Discussion by PoByBolek** [amzn/selling-partner-api-models#3535] [General] [answered]
I'm trying to request the `GET_LEDGER_DETAIL_VIEW_DATA` report but keep getting `403 Forbidden` errors. I'm somewhat convinced that this is a role problem because I _can_ request other reports like the `GET_SALES_AND_TRAFFIC_REPORT` for example, which requires the "Brand Analytics" role in Seller Central.

I tried enabling basically every other role for my app in Seller Central without success. I still keep getting `403 Forbidden` responses with this error message:
```json
{
  "errors": [
    {
      "code": "Unauthorized",
      "message": "Access to the resource is forbidden",
      "details": ""
    }
  ]
}
```

**Comment by sungolivia** [comment, marked as answer]
Looks like this role does! https://developer-docs.amazon.com/sp-api/docs/amazon-fulfillment-role#feeds-reports-and-notifications

**Reply by PoByBolek** [reply]
Thanks! I didn't notice that the role definitions list the individual reports and feeds to which they grant access.

Also, it looks like I just had to wait a bit after assigning the role to my app before I could actually make a successful request.
